websites/H/HTB Academy/presence.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-one-time-vars/no-one-time-vars */
+const presence = new Presence({
+	clientId: "1200517025383075840",
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000),
+path = document.location.pathname;
```

Comment:
no. This way the path won't work in the correct way

Reply:
It does :D